src/core/a-entity.js

```diff
@@ -78,11 +84,15 @@ var proto = Object.create(ANode.prototype, {
 
   applyMixin: {
     value: function (attr) {
-      var attrValue = this.getAttribute(attr);
+      var attrValue;
       if (!attr) {
         this.updateComponents();
         return;
       }
+      attrValue = this.getAttribute(attr);
+      // turn null into undefined because getAttribute
```

Comment:
can 1line `Make absence of attribute for getAttribute return undefined rather than null`.